src/raft_server.c

```diff
@@ -523,12 +523,16 @@ int raft_already_voted(raft_server_t* me_)
 
 static int __should_grant_vote(raft_server_private_t* me, msg_requestvote_t* vr)
 {
+    raft_node_t *my_node = raft_get_my_node((void*)me);
+
+    assert(NULL != my_node);
```

Comment:
Similarly, is it possible for a new replica with an empty membership to receive a RV request? E.g., the leader commits the nonvoting entry and voting entry on majorities that do not include this new replica, and crashes. Now anther voting node attempts to become the new leader, sending RVs to all voting nodes, which includes this new replica. Then we'll fail this assertion, won't we?